EDU-746 - REVIEW: Pickup Availability documentation (#19)
vtex-apps/pickup-availability: docs/README.md

>ℹ️ This component uses the [`ProductContext`](https://developers.vtex.com/docs/apps/vtex.product-context) app and should be implemented inside the `store.product` block. 

## Before you begin

Make sure you have configured in-store pickup for your store, including pickup points and shipping policies.

## Installation

1. Open your [Store theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme) project in a code editor.
2. In the terminal, [install](https://developers.vtex.com/docs/guides/vtex-io-documentation-installing-an-app) the `pickup-availability` app in your account by running the `vtex install vtex.pickup-availability` command.
3. Add the `pickup-availability` app to your theme's `manifest.json` file under `peerDependencies`:

  ```json
  "peerDependencies": {
    	"vtex.pickup-availability": "0.x"
  }
  ```
## Configuration

Declare the `pickup-availability` block in the desired product template, such as `store.product`. For example:

```json
"store.product": {
  "children": [
      "flex-layout.row#product-main",
      "flex-layout.row#description",
      "shelf.relatedProducts",
      "pickup-availability"
    ]
 },
```

## Customization

> To apply CSS customizations on this and other blocks, you can use the following CSS Handles. For detailed instructions, see the guide [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).

Reviewer: ```suggestion
To apply CSS customizations to this and other blocks, follow the instructions in [Using CSS Handles for store customization](https://developers.vtex.com/docs/guides/vtex-io-documentation-using-css-handles-for-store-customization).
```
